lib/api/core/janitor.js

```diff
@@ -362,6 +465,109 @@ class Janitor {
     process.exit(0);
   }
 
+  _loadJson (filePath) {
+    return new Bluebird((resolve, reject) => {
+      fs.readFile(filePath, 'utf8', (error, rawData) => {
+        if (error) {
+          return reject(error);
+        }
+
+        try {
+          const data = JSON.parse(rawData);
+
+          resolve(data);
+        }
+        catch (e) {
+          reject(e);
+        }
+      });
+    });
+  }
+
+  _createRoles (roles) {
+    if (! roles) {
+      return Bluebird.resolve();
+    }
+
+    const promises = Object.keys(roles).map(roleId => {
+      const request = new Request({
+        controller: 'security',
+        action: 'createOrReplaceRole',
+        refresh: 'wait_for',
+        _id: roleId,
+        body: roles[roleId]
+      });
+
+      return this.kuzzle.funnel.processRequest(request);
+    });
+
+    return Bluebird.all(promises);
+  }
+
+  _createProfiles (profiles) {
+    if (! profiles) {
+      return Bluebird.resolve();
+    }
+
+    const promises = Object.keys(profiles).map(profileId => {
+      const request = new Request({
+        controller: 'security',
+        action: 'createOrReplaceProfile',
+        refresh: 'wait_for',
+        _id: profileId,
+        body: profiles[profileId]
+      });
+
+      return this.kuzzle.funnel.processRequest(request);
+    });
+
+    return Bluebird.all(promises);
+  }
+
+  _createUsers (users) {
+    if (! users) {
+      return Bluebird.resolve();
+    }
+
+    const promises = Object.keys(users).map(userId => {
+      const request = new Request({
+        controller: 'security',
+        action: 'createUser',
+        refresh: 'wait_for',
+        _id: userId,
+        body: users[userId]
+      });
+
+      return this.kuzzle.funnel.processRequest(request);
+    });
+
+    return Bluebird.all(promises);
+  }
+
+  _deleteUsers (users) {
+    if (! users) {
+      return Bluebird.resolve();
+    }
+
+    const
+      ids = Object.keys(users).map(userId => userId),
+      request = new Request({
+        controller: 'security',
+        action: 'mDeleteUsers',
+        refresh: 'wait_for',
+        body: { ids }
+      });
+
+    return this.kuzzle.funnel.processRequest(request)
+      .catch(error => {
+        if (error.status === 206) {
+          return Bluebird.resolves();
```

Comment:
`Bluebird.resolves` is not a function. This still works because you end up returning `undefined` instead of a new empty promise resolution, which leads to the same result. 
Replace with `return null;`